### DISCUSSION TITLE
Eclair architecture

### DISCUSSION THREAD

**Discussion by kiwiidb** [ACINQ/eclair#2582] [Q&A] [answered]
On a high level, how does the architecture of Eclair make it possible to run a single node in a distributed way, and especially what are the differences with LND's (or CLN's) architecture where it seems that the only good option of doing HA is by using a phantom node + multiple, seperate real nodes?

**Comment by t-bast** [comment, marked as answer]
I recommend having a look at our [cluster documentation](https://github.com/ACINQ/eclair/blob/master/docs/Cluster.md) and [architecture blog post](https://medium.com/@ACINQ/the-architecture-behind-eclair-ca0c99880962), which should contain details about eclair's setup. If you have questions about specific details of this setup, don't hesitate to add them and we'll dive more deeply into it.
